container-host-files/etc/scf/config/opinions.yml

```diff
@@ -40,6 +40,78 @@ properties:
     user: admin
     org: test-brain-org
     space: test-brain-space
+  autoscaler:
+    api_server:
+      db_config:
+        idle_timeout: 100
+        max_connections: 10
+        min_connections: 0
+      info:
+        build: "beta"
+        description: "Provides autoscaling capabilities based on policies, for apps deployed on Cloud Foundry"
+        name: "SCF App Autoscaler"
```

Comment:
Hmm, we'll need to figure out how to make white-labelling this easier in the future.

Of course, that might just be ops files…

Reply:
We could expose them in the manifest again if they're ever required.